plugins/embed-optimizer/tests/test-optimization-detective.php

```diff
@@ -122,5 +123,7 @@ static function ( $matches ) {
 			$this->remove_initial_tabs( $buffer ),
 			"Buffer snapshot:\n$buffer"
 		);
+
+		unset( $GLOBALS['template'] );
```

Comment:
This should rather be done in a `tear_down` method, since if there is an error in a test then this wouldn't be run. Same for `test-helper.php`.